### ISSUE TITLE
series_fmt in custom_series notebook

### ISSUE THREAD

**Issue by lambdaofgod** [stared/livelossplot#49]
The example from [custom_series notebook](https://github.com/stared/livelossplot/blob/master/examples/custom_series.ipynb) doesn't work:

> TypeError: __init__() got an unexpected keyword argument 'series_fmt'

Was series_fmt parameter in an older version?

**Comment by stared** [comment]
It was introduced in #46. @sebastienlange, could you look at it?

**Comment by sebastienlange** [comment]
@lambdaofgod, it looks like jupyter notebook has a different version of livelossplot in memory. 
Could you try restarting your jupyter notebook?

@stared  I just downloaded a fresh livelossplot and tried custom_series notebook and minimal.
Both gave me the same different error and I had to  install neptune-cli. Is it normal?
Then both worked for me.

**Comment by lambdaofgod** [comment]
@sebastienlange yes I did try restarting notebook. I also made sure that I actually had newest version of livelossplot.

**Comment by sebastienlange** [comment]
@lambdaofgod  on my computer, after installation, PlotLosses class is located at C:\Users\lange\Anaconda3\Lib\site-packages\livelossplot\generic_plot.py. 
Could you try to located the same file on your configuration and share the __init__ def ?

Should be:
def \_\_init\_\_(self,
                 figsize=None,
                 cell_size=(6, 4),
                 dynamic_x_axis=False,
                 max_cols=2,
                 max_epoch=None,
                 metric2title={},
                 **series_fmt={'training':** '{}', **'validation':'val_{}'},**
                 validation_fmt="val_{}",
                 plot_extrema=True,
                 fig_path=None,
                 target=MATPLOTLIB_TARGET):
...

**Comment by stared** [comment]
@sebastienlange It should not require `neptune` (after #47 it required, but I fixed that with 7876f0690e1d4fe229ceecd696d314cb73d79855; as the general philosophy is to avoid ML packages). Is this still a problem? (If so, another Issue would be on spot.)

**Comment by stared** [comment]
No updates - or confirmations - I close this issue.
